[GTK][WPE] Build fails with ORIENTATION_EVENTS enabled

clopez opened the pull request:
#### 2510901a9a26f1f11daf5e9ae29382dce1fe7f0c
<pre>
[GTK][WPE] Build fails with ORIENTATION_EVENTS enabled
<a href="https://bugs.webkit.org/show_bug.cgi?id=242637">https://bugs.webkit.org/show_bug.cgi?id=242637</a>

Reviewed by Michael Catanzaro.

The function WebChromeClient::deviceOrientation() is only
defined for the iOS build. Fix the build failure for other
platforms by defining a skeleton function.

* Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp:
(WebKit::WebChromeClient::deviceOrientation const):

Canonical link: <a href="https://commits.webkit.org/252385@main">https://commits.webkit.org/252385@main</a>
</pre>

mcatanzaro reviewed (approved):
Does device orientation actually work if this function is not properly implemented? Probably not?

If it works, we should enable the build option unconditionally. If it doesn't work, I'm not quite sure what you're accomplishing, but I suppose allowing the build to succeed is better than not....

clopez committed:
[GTK][WPE] Build fails with ORIENTATION_EVENTS enabled
https://bugs.webkit.org/show_bug.cgi?id=242637

Reviewed by Michael Catanzaro.

The function WebChromeClient::deviceOrientation() is only
defined for the iOS build. Fix the build failure for other
platforms by defining a skeleton function.

* Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp:
(WebKit::WebChromeClient::deviceOrientation const):

Canonical link: https://commits.webkit.org/252385@main
